app/controllers/concerns/authorizable.rb

```diff
@@ -13,6 +13,18 @@ def ensure_authenticated
     return render_error status: :unauthorized unless current_user
   end
 
+  # PermissionsChecker service will return a true or false depending on whether the current_user's role has the provided permission_name
+  def ensure_authorized(permission_name)
+    # TODO: - should this return a status: :not_found instead of :unauthorized if the user doesn't have the permission AND the param doesn't exists?
+    return render_error status: :unauthorized unless PermissionsChecker.new(
```

Comment:
Isn't the status `forbidden` more adequate  than `unauthorized` or we're obscuring the responses?

Reply:
Why would we obscure this response?
I was hesitating between `unauthorized` and `not_found`